Neos.Flow/Classes/Http/Cookie.php

```diff
@@ -86,6 +90,23 @@ class Cookie
      */
     protected $httpOnly;
 
+    /**
+     * Possible values: none, lax, or strict (RFC 6265bis-05, 8.8)
+     *
+     * sameSite=strict
+     *   Cookie will only be sent in a first-party context, cookie is ignored on the initial request to your site.
+     *   This is a good setting when you have cookies relating to functionality, such as changing a password
+     * sameSite=strict
+     *   Cookie will only be sent in a top-level navigations.
+     *   This is a good setting when you need the cookie on the initial request, such as session login via cookie
+     * sameSite=none
+     *   Cookie will sent in a third-party context.
+     *   This is a good setting when you need the cookie in cors ajax request, such as providing an api with session login via cookie
```

Comment:
Shouldn't one description be for `lax`?

Reply:
Indeed! Eagle-eye :)

Reply:
Yes that was a nice catch. Will fix tomorrow